### PR TITLE
complete nc-news-7

### PR DESCRIPTION
**Pull request by haydenexley**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -192,6 +192,46 @@ describe("app", () => {
...
+        .send(newComment)
+        .expect(201)
+        .then(({ body: { comment } }) => {
+          expect(comment[0]).toHaveProperty("comment_id", 19);
+          expect(comment[0]).toHaveProperty("body", "cool stuff!");
+          expect(comment[0]).toHaveProperty("article_id", 2);
+          expect(comment[0]).toHaveProperty("author", "rogersop");
+          expect(comment[0]).toHaveProperty("votes", 0);
+          expect(comment[0]).toHaveProperty("created_at", expect.any(String));
+        });
```
Comment (EddTheDuck) [inline __tests__/app.test.js:209]: Need to add a test for a 201 - ignores unnecessary properties

Comment (EddTheDuck) [reply __tests__/app.test.js:209]: Also missing a test for a 400 invalid ID, e.g. string of "not-an-id"

**Review by EddTheDuck — CHANGES_REQUESTED** [review]
Looking good however you are missing a couple of tests, please add these then submit a quick PR. Ta!

**Review by nc-poppy-m — APPROVED** [review]
Superb! Good to go

**Pull request merged**